src/views/Farms/components/Table/Actions/ActionPanel.tsx

```diff
@@ -0,0 +1,162 @@
+import getLiquidityUrlPathParts from 'utils/getLiquidityUrlPathParts'
+import { communityFarms } from 'config/constants'
+import { CommunityTag, CoreTag, DualTag } from 'components/Tags'
+
+import HarvestAction from './HarvestAction'
+import StakedAction from './StakedAction'
+import Apr, { AprProps } from '../Apr'
+import Multiplier, { MultiplierProps } from '../Multiplier'
+import Liquidity, { LiquidityProps } from '../Liquidity'
+
+export interface ActionPanelProps {
+  apr: AprProps
+  multiplier: MultiplierProps
+  liquidity: LiquidityProps
+  details: FarmWithStakedValue
+}
+
+const Container = styled.div`
+  background: ${(props) => props.theme.colors.background};
+  display: flex;
+  width: 100%;
+  flex-direction: column-reverse;
+  padding: 24px;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    flex-direction: row;
+    padding: 16px 32px;
+  }
+`
+
+const StyledLinkExternal = styled(LinkExternal)<{ svg?: boolean }>`
+  font-weight: 400;
+  margin-top: 8px;
+
+  svg {
+    display: ${(props) => (props.svg ? 'block' : 'none')};
+  }
+`
+
+const StakeContainer = styled.div`
+  color: ${(props) => props.theme.colors.text};
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+
+  a {
+    margin-top: 0;
+    margin-left: 8px;
+  }
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    justify-content: flex-start;
+  }
+`
+
+const TagsContainer = styled.div`
+  display: flex;
+  align-items: center;
+  margin-top: 25px;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    margin-top: 16px;
+  }
+
+  > div {
+    height: 24px;
+    padding: 0 6px;
+    font-size: 14px;
+    margin-right: 4px;
+
+    svg {
+      width: 14px;
+    }
+  }
+`
+
+const ActionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    flex-direction: row;
+    align-items: center;
+    flex-grow: 1;
+    flex-basis: 0;
+  }
+`
+
+const InfoContainer = styled.div`
+  min-width: 200px;
+`
+
+const ValueContainer = styled.div`
+  display: block;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    display: none;
+  }
+`
+
+const ValueWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 4px 0px;
+`
+
+const ActionPanel: React.FunctionComponent<ActionPanelProps> = ({ details, apr, multiplier, liquidity }) => {
+  const farm = details
+
+  const TranslateString = useI18n()
+  const { quoteTokenAdresses, quoteTokenSymbol, tokenAddresses, tokenSymbol, dual } = farm
+  const lpLabel = farm.lpSymbol && farm.lpSymbol.toUpperCase().replace('PANCAKE', '')
+  const liquidityUrlPathParts = getLiquidityUrlPathParts({ quoteTokenAdresses, quoteTokenSymbol, tokenAddresses })
+  const bsc = `https://bscscan.com/address/${farm.lpAddresses[process.env.REACT_APP_CHAIN_ID]}`
+  const info = `https://pancakeswap.info/pair/${farm.lpAddresses[process.env.REACT_APP_CHAIN_ID]}`
```

Comment:
Please put the chain id in a variable at the top of the file if we are using it more than once.